app/views/RoomActionsView/index.js

```diff
@@ -412,6 +414,79 @@ class RoomActionsView extends React.Component {
 		);
 	}
 
+	handleLeaveTeam = async(selected) => {
+		const { room } = this.state;
+		try {
+			const { navigation, isMasterDetail } = this.props;
+			const result = await RocketChat.leaveTeam({ teamName: room.name });
+
+			if (selected) {
+				try {
+					selected.map(item => RocketChat.leaveRoom(item.rid, item.t));
```

Comment:
`teams.leave` accepts a `rooms` option that does it already.
https://github.com/RocketChat/Rocket.Chat/blob/develop/app/api/server/v1/teams.ts#L316